app-modules/service-management/database/migrations/2025_05_12_101058_tmp_data_seed_for_survey_response_in_service_request_type_email_templates.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        DB::transaction(function () {
+            $table = 'service_request_type_email_templates';
+            $chunkSize = 100;
+            DB::table($table)
+                ->where('type', 'survey_response')
+                ->whereIn('role', ['auditor', 'manager'])
+                ->orderBy('id')
+                ->chunkById($chunkSize, function ($records) use ($table) {
+                    foreach ($records as $record) {
+                        DB::table($table)->where('id', $record->id)->delete();
+                    }
+                });
```

Comment:
If all you are doing is deleting all of these, you do not need to chunk. You can just call `delete()` on line `50`.

The database will take care of deleting all the records for you then.